CHANGELOG.adoc

```diff
@@ -122,6 +122,14 @@ This incorporates the following executed proposals:
 
 Added the ic-btc-adapter binary to the cache.
 
+= 0.9.4
+
+== DFX
```

Comment:
There's already a section for dfx in the new version, which will be dfx 0.10.0

Reply:
I see now - updated with the correct spot

Reply:
It's still kind of in an odd place - `===` under `== Replica`